src/sentry_plugins/github/webhooks/events/pull_request.py

```diff
@@ -53,16 +54,13 @@ def __call__(self, event, organization):
             )
             author_email = commit_author.email
         except CommitAuthor.DoesNotExist:
-            try:
-                user_model = User.objects.filter(
-                    social_auth__provider="github",
-                    social_auth__uid=user["id"],
-                    org_memberships=organization,
-                )[0]
-            except IndexError:
-                pass
-            else:
-                author_email = user_model.email
+            for social_auth in UserSocialAuth.objects.filter(provider="github", uid=user["id"]):
```

Comment:
`UserSocialAuth` is a control silo model. We would need to fetch social auth data either from the request's existing User record, or make another RPC call to get that data.